src/app/meshes/views/MeshListView.vue

```diff
@@ -1,124 +1,123 @@
 <template>
-  <RouteView
-    v-slot="{ route }"
-    name="mesh-list-view"
+  <DataSource
+    v-slot="{ data: me }: MeSource"
+    src="/me"
   >
-    <DataSource
-      v-slot="{data, error}: MeshCollectionSource"
-      :src="`/meshes?page=${props.page}&size=${props.size}`"
+    <RouteView
+      v-if="me"
+      v-slot="{ route, t }"
+      name="mesh-list-view"
+      :params="{
+        page: 1,
+        size: me.pageSize
+      }"
```

Comment:
This is the best place to have a look at what this looks like at an application level.

Here we:

1. Load in any user values (eventually from local storage) to use as defaults.
2. Set some default params on `RouteView#params`. If the URL doesn't have things then we `route.replace` them onto it (i.e. we don't add that navigation to history)
3. Under the hood if the URL has these params but they aren't numerical (I'll be adding more simple validation for other things later), then we again fallback to the default.
4. Below this comment we swap out the previous `props.page` for `route.params.page` to take advantage of all this. Most of the green below is because i wrapped the entire thing in a DataSource